src/EmergencyMultisig.sol

```diff
@@ -59,6 +59,7 @@ contract EmergencyMultisig is IEmergencyMultisig, IMembership, PluginUUPSUpgrade
     struct MultisigSettings {
         bool onlyListed;
         uint16 minApprovals;
```

Comment:
The standard multisig is the source of truth regarding the address list, whereas the emergency multisig uses the standard multisig's list. This is to save storage and transactions to maintain, assuming that both had the same members.

Reply:
I don't follow. But I can ask later in our next call.

Reply:
What I mean is that instead of having two separate lists that need to be maintained separately, we only have one and the standard multisig acts as the database. 

This is assuming that both plugins are controlled by the same group of people. If this wasn't the case, then we would need to undo this change and use two almost identical multisigs, but with independent settings/signers.

Reply:
Ah, I see, so `addresslistSource` in EmergencyMultisig.sig points to the Multisig contract which manages the actual list of approvers, right?

Reply:
@dantaik correct
We can improve the naming if this feels confusing, but it basically points to the address list to use